UNRELEASED.md

```diff
@@ -60,6 +60,19 @@ following warning will be shown during compilation or in the IDE:
 
 For more information, see [Reference: Templates: Template-local Definitions (Deprecated)](https://docs.daml.com/2.8.0/daml/reference/templates.html#template-local-definitions-deprecated)
 
+## Removal of deprecated 'controller..can' syntax
+
+The 'controller..can' syntax for defining template choices, deprecated since
+Daml 2.0, has now been completely removed, so projects that use them will no
+longer be accepted. Those choices should instead be defined using 'choice-first'
+syntax.
+
+Note that, as a consequence, the warning flags `-Wcontroller-can` and
+`-Wnocontroller-can` are not accepted anymore.
+
+See [Deprecation of `controller`-first syntax: Migrating](https://docs.daml.com/2.7.0/daml/reference/choices.html#migrating)
```

Comment:
intentionally pointing to the 2.___seven___.0 docs, I don't think we want to keep the migration guide in the 2.8.0 docs; if we do, we'd just be pushing the deprecation down the line